File: app/src/main/java/com/siercuit/cartelera/views/IndicatorView.java
package com.siercuit.cartelera.views;

import android.content.Context;
import android.util.AttributeSet;

import com.siercuit.cartelera.R;

public class IndicatorView extends FontIconView
{
    public IndicatorView(Context context) {
        super(context);
    }

    public IndicatorView(Context context, AttributeSet attrs)
    {
        super(context, attrs);
    }

    public void setState(Boolean opened)
    {
        if (opened) {
            setText(R.string.fi_minus_round);
        } else {
            setText(R.string.fi_plus_round);
        }
    }

}

File: app/src/main/kotlin/io/vextil/billboard/ui/IndicatorView.kt
package io.vextil.billboard.ui

import android.content.Context
import android.util.AttributeSet
import io.vextil.billboard.ui.FontIconView

import io.vextil.billboard.R

class IndicatorView : FontIconView {

    constructor(context: Context) : super(context)

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs)

    fun setState(opened: Boolean) {
        if (opened) {
            setText(R.string.fi_minus_round)
        } else {
            setText(R.string.fi_plus_round)
        }
    }

}
